mapping_parenting_tech/analysis/prototyping/clustering_descriptions.py

```diff
@@ -42,11 +43,18 @@
 
 # %%
 # Load in the descriptions
-descriptions = pd.read_csv(open(INPUT_DATA / "simple_app_descriptions.csv", "rb"))
+details = pd.read_json(INPUT_DATA / "all_app_details3.json", orient="index")
```

Comment:
There is no file `all_app_details3.json` - did you perhaps mean `all_app_details.json` ?

Reply:
I used run the rest of the notebook's code, I used `all_app_details.json`

Reply:
Nothing says 'in development' like a number at the end of a filename... 'all_app_details83_test_final'... (it's sorted now :) )